openwisp_controller/subnet_division/rule_types/vpn.py

```diff
@@ -51,3 +52,14 @@ def post_provision_handler(instance, provisioned, **kwargs):
             instance.ip = provisioned['ip_addresses'][0]
             instance.full_clean()
             instance.save()
+
+    @classmethod
+    def destroy_provisioned_subnets_ips(cls, instance, **kwargs):
```

Comment:
Is there already code calling this method? I assume so because I don't see any new line calling this.